resources/shared_translations.txt

```diff
@@ -4956,6 +4956,8 @@
 "operations.tree.yleisten-alueiden-luvat" "en" "Public areas (placement agreement, street permit, intended use of areas)"
 "operations.tree.yleisten-alueiden-luvat" "fi" "Yleiset alueet (Sijoittamissopimus, katulupa, alueiden käyttö)"
 "operations.tree.yleisten-alueiden-luvat" "sv" "Användning av allmänna områden (avtal om placering, gatutillstånd, användning av område)"
+"operations.tree.ymparistoluvan-selventaminen" "en" "Clarification of environmental permit"
```

Comment:
Tämä on sinällään Lupiksen kantilta tyystin merkityksetöntä, mutta jäin miettimään pitäisikö tämän olla joko Environmental permit clarification tai Clarification of an environmental permit 🤷